kedro-datasets/kedro_datasets/tensorflow/tensorflow_model_dataset.py

```diff
@@ -50,14 +50,14 @@ class TensorFlowModelDataset(AbstractVersionedDataset[tf.keras.Model, tf.keras.M
         >>> import tensorflow as tf
         >>> import numpy as np
         >>>
-        >>> dataset = TensorFlowModelDataset(tmp_path / "data/06_models/tensorflow_model.h5")
-        >>> model = tf.keras.Model()
-        >>> predictions = model.predict([...])
+        >>> dataset = TensorFlowModelDataset(filepath=tmp_path / "data/06_models/tensorflow_model.h5")
+        >>> model = tf.keras.Sequential([tf.keras.layers.Dense(5, input_shape=(3,)),tf.keras.layers.Softmax()])
+        >>>
+        # >>> x = tf.random.uniform((10, 3))
+        # >>> predictions = model.predict(x)
```

Comment:
It looks weird to comment these lines out like that, what about
```suggestion
        >>> # x = tf.random.uniform((10, 3))
        >>> # predictions = model.predict(x)
```